sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/spanner/MutationUtils.java

```diff
@@ -0,0 +1,21 @@
+package org.apache.beam.sdk.io.gcp.spanner;
```

Comment:
Please add license headers to make `./gradlew :rat` happy. Alternatively move `isPointDelete` into `MutationCellCounter` - as it's only used from there.

Reply:
Done